docs/guide/limitations.md

```diff
@@ -1,16 +1,18 @@
 # Limitations
 
-Limitations are crucial building blocks of the permissions system in eZ Platform. They provide the restrictions you can apply to a given access right to limit the right according to certain conditions.
+Limitations are part of the permissions system.
+They limit the access granted to users by [Policies](permissions.md#permission-overview).
```

Comment:
Please explain this further as it may confuse users. 
Limitations grant access to defined Locations/Content Types etc. Limitation specifies what users can do, not what they can't do.